src/stories/Image/Image.style.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const ImageContainer = styled.div`
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  justify-content: center;
+`;
+
+export const ImageStyle = styled.img`
+  height: 9.3rem;
```

Comment:
why need height here @PRABgit01

Reply:
to make image smaller because it was taking same size of container which i have made